### PR TITLE
Dev server

### PR DESCRIPTION
**Pull request by ericelliott**
* Share the main render route with the dev server.
* Upgrade `devServer.js` to ES6

### CODE REVIEW

**Comment by ericelliott** [comment]
Ping @nkbt @vasco3

File: source/client/index.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { render } from 'react-dom';
 import createApp from 'shared/components/app';
 
 const App = createApp(React);
-const props = {
-  title: 'Your App'
+const props = window.payload || {
```
Comment (nkbt) [inline source/client/index.js]:
```
const {...props = {title: 'Default client title'}} = global.window.payload
```

=)

yeah, i'm bit obsessed with destructuring...

**Comment by nkbt** [comment]
Looks good.

**Comment by ericelliott** [comment]
:+1:

**Pull request merged**